doc/run_experiment.rst

```diff
@@ -406,6 +406,26 @@ example, if you wanted to collapse the labels ``beagle`` and ``dachsund`` into a
 
 Any labels not included in the dictionary will be left untouched.
 
+One other use case for ``class_map`` is to deal with classification labels that
+would be converted to ``float`` improperly. All ``Reader`` sub-classes use the
+``safe_float`` function internally to read labels. This function tries to
```

Comment:
Please link `safe_float` to the API documentation. You should just be able to replace `safe_float` with :py:mod:`skll.data.readers.safe_float`.